aries-site/src/pages/index.js

```diff
@@ -13,8 +25,41 @@ const Index = () => {
     }
   }, []);
 
-  // return <Landing />;
-  return <About />;
+  return (
+    <PageLayout title="Home" isLanding>
+      {/* <Grid columns={['flex', 'xxlarge', 'flex']}> */}
+      {/* <Box /> */}
+      <Box gap="large">
+        <HomeTiles>
+          <Tile background="white">
+            <Image src="/image-hands.png" />
```

Comment:
```suggestion
            <Image src="/image-hands.png" fit="cover" />
```
We should apply fit, otherwise the proportions of the image get squished on small screen sizes.